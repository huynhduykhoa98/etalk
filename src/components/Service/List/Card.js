/* eslint-disable linebreak-style */
import React, { Component } from 'react';
import { FormattedMessage } from 'react-intl';

class Card extends Component {
  render() {
    return (
      <li>
        <img src={this.props.img} alt={this.props.maintext} width={100} />
        <div>
          <h3>
            <FormattedMessage
              id="serviceHeader"
              values={{ order: `${this.props.value}` }}
            />
          </h3>
          <FormattedMessage
            id="serviceText"
            values={{
              order: `${this.props.value}`,
              p: (chunks) => <p>{chunks}</p>,
            }}
          />
        </div>
      </li>
    );
  }
}

export default Card;
