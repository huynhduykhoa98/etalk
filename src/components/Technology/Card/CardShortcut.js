/* eslint-disable linebreak-style */
import React, { Component } from 'react';

class CardShortcut extends Component {
  render() {
    return (
      <a href={`${this.props.href}`} title={`${this.props.title}`}>
        <img src={`${this.props.img}`} alt={`${this.props.title}`} />
        {`${this.props.title}`}
      </a>
    );
  }
}

export default CardShortcut;
