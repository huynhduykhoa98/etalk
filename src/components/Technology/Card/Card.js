/* eslint-disable linebreak-style */
/* eslint-disable class-methods-use-this */
import React, { Component } from 'react';
import CardShortcut from './CardShortcut';

class Card extends Component {
  render() {
    return (
      <div className="pic">
        <CardShortcut
          href="//www.w3.org/standards/webdesign/htmlcss"
          title="GSAP"
          img="https://spider.vn/images/icon_gsap.svg"
        />
        <CardShortcut
          href="//www.w3.org/standards/webdesign/htmlcss"
          title="HTML & CSS"
          img="https://spider.vn/images/icon_html.png"
        />

        <CardShortcut
          href="//www.w3.org/standards/webdesign/script"
          title="JavaScript"
          img="https://spider.vn/images/icon_js.png"
        />

        <CardShortcut
          href="//babeljs.io"
          title="Babel"
          img="https://spider.vn/images/icon_babel.svg"
        />

        <CardShortcut
          href="//pugjs.org"
          title="Pug"
          img="https://spider.vn/images/icon_pugjs.svg"
        />

        <CardShortcut
          href="//sass-lang.com"
          title="Sass"
          img="https://spider.vn/images/icon_sass.svg"
        />

        <CardShortcut
          href="//getbootstrap.com"
          title="Bootstrap"
          img="https://spider.vn/images/icon_bootstrap.png"
        />

        <CardShortcut
          href="//material.io"
          title="Material"
          img="https://spider.vn/images/icon_material.png"
        />

        <CardShortcut
          href="//jquery.com"
          title="jQuery"
          img="https://spider.vn/images/icon_jquery.svg"
        />

        <CardShortcut
          href="//nextjs.org"
          title="Next.js"
          img="https://spider.vn/images/icon_nextjs.png"
        />

        <CardShortcut
          href="//angular.io"
          title="Angular"
          img="https://spider.vn/images/icon_angular.png"
        />

        <CardShortcut
          href="//reactjs.org"
          title="React"
          img="https://spider.vn/images/icon_react.svg"
        />

        <CardShortcut
          href="//redux.js.org"
          title="Redux"
          img="https://spider.vn/images/icon_redux.svg"
        />

        <CardShortcut
          href="//graphql.org"
          title="GraphQL"
          img="https://spider.vn/images/icon_graphql.svg"
        />

        <CardShortcut
          href="//vuejs.org"
          title="Vue"
          img="https://spider.vn/images/icon_vue.png"
        />

        <CardShortcut
          href="//lodash.com"
          title="Lodash"
          img="https://spider.vn/images/icon_lodash.svg"
        />

        <CardShortcut
          href="//underscorejs.org"
          title="Underscore"
          img="https://spider.vn/images/icon_underscore.png"
        />

        <CardShortcut
          href="//backbonejs.org"
          title="Backbone.js"
          img="https://spider.vn/images/icon_backbone.png"
        />

        <CardShortcut
          href="//emberjs.com"
          title="Ember.js"
          img="https://spider.vn/images/icon_ember.png"
        />

        <CardShortcut
          href="//phaser.io"
          title="Phaser"
          img="https://spider.vn/images/icon_phaser.png"
        />

        <CardShortcut
          href="//threejs.org"
          title="Three.js"
          img="https://spider.vn/images/icon_threejs.png"
        />

        <CardShortcut
          href="//d3js.org"
          title="D3.js"
          img="https://spider.vn/images/icon_d3.png"
        />

        <CardShortcut
          href="//www.gatsbyjs.org"
          title="Gatsby"
          img="https://spider.vn/images/icon_gatsby.svg"
        />

        <CardShortcut
          href="//www.webcomponents.org"
          title="Web Components"
          img="https://spider.vn/images/icon_web_components.svg"
        />

        <CardShortcut
          href="//www.polymer-project.org"
          title="Polymer"
          img="https://spider.vn/images/icon_polymer.svg"
        />

        <CardShortcut
          href="//developers.google.com/web/progressive-web-apps"
          title="Progressive Web Apps"
          img="https://spider.vn/images/icon_web_app.png"
        />

        <CardShortcut
          href="//serverless.css-tricks.com"
          title="Serverless"
          img="https://spider.vn/images/icon_serverless.svg"
        />

        <CardShortcut
          href="//headlesscms.org/"
          title="Headless CMS"
          // img="https://spider.vn/images/icon_headlesscms.svg"
        />

        <CardShortcut
          href="//www.php.net"
          title="PHP"
          img="https://spider.vn/images/icon_php.svg"
        />

        <CardShortcut
          href="//laravel.com"
          title="Laravel"
          img="https://spider.vn/images/icon_laravel.svg"
        />

        <CardShortcut
          href="//octobercms.com"
          title="October"
          img="https://spider.vn/images/icon_october.png"
        />

        <CardShortcut
          href="//symfony.com"
          title="Symfony"
          img="https://spider.vn/images/icon_polymer.svg"
        />

        <CardShortcut
          href="//www.yiiframework.com"
          title="Yii"
          img="https://spider.vn/images/icon_yii.svg"
        />

        <CardShortcut
          href="//www.codeigniter.com"
          title="CodeIgniter"
          img="https://spider.vn/images/icon_codeigniter.svg"
        />

        <CardShortcut
          href="//wordpress.org"
          title="WordPress"
          img="https://spider.vn/images/icon_wordpress.svg"
        />

        <CardShortcut
          href="//woocommerce.com"
          title="WooCommerce"
          img="https://spider.vn/images/icon_woocommerce.svg"
        />

        <CardShortcut
          href="//www.drupal.org"
          title="Drupal"
          img="https://spider.vn/images/icon_drupal.svg"
        />

        <CardShortcut
          href="//golang.org"
          title="Golang"
          img="https://spider.vn/images/icon_golang.svg"
        />

        <CardShortcut
          href="//cordova.apache.org"
          title="Cordova"
          img="https://spider.vn/images/icon_cordova.svg"
        />

        <CardShortcut
          href="//ionicframework.com"
          title="Ionic"
          img="https://spider.vn/images/icon_ionic.svg"
        />

        <CardShortcut
          href="//facebook.github.io/react-native"
          title="React Native"
          img="https://spider.vn/images/icon_react.svg"
        />

        <CardShortcut
          href="//www.nativescript.org"
          title="NativeScript"
          img="https://spider.vn/images/icon_nativescript.svg"
        />

        <CardShortcut
          href="//flutter.dev"
          title="Flutter"
          img="https://spider.vn/images/icon_flutterio.svg"
        />

        <CardShortcut
          href="//electronjs.org"
          title="Electron"
          img="https://spider.vn/images/icon_electronjs.svg"
        />

        <CardShortcut
          href="//nodejs.org"
          title="Node.js"
          img="https://spider.vn/images/icon_nodejs.svg"
        />

        <CardShortcut
          href="//expressjs.com"
          title="Express"
          // img="https://spider.vn/images/expressjs.svg"
        />

        <CardShortcut
          href="//hapi.dev"
          title="Hapi"
          img="https://spider.vn/images/icon_hapi.svg"
        />

        <CardShortcut
          href="//sailsjs.com"
          title="Sails"
          img="https://spider.vn/images/icon_sails.svg"
        />

        <CardShortcut
          href="//sequelize.org"
          title="Sequelize"
          img="https://spider.vn/images/icon_sequelizejs.svg"
        />

        <CardShortcut
          href="//www.meteor.com"
          title="Meteor"
          img="https://spider.vn/images/icon_meteor.svg"
        />

        <CardShortcut
          href="//www.sqlite.org"
          title="SQLite"
          img="https://spider.vn/images/icon_sqlite.svg"
        />

        <CardShortcut
          href="//www.mysql.com"
          title="MySQL"
          img="https://spider.vn/images/icon_mysql.svg"
        />

        <CardShortcut
          href="//www.postgresql.org"
          title="PostgreSQL"
          img="https://spider.vn/images/icon_postgresql.svg"
        />

        <CardShortcut
          href="//www.mongodb.com"
          title="MongoDB"
          img="https://spider.vn/images/icon_mongodb.svg"
        />

        <CardShortcut
          href="//redis.io"
          title="Redis"
          img="https://spider.vn/images/icon_redis.svg"
        />

        <CardShortcut
          href="//www.npmjs.com"
          title="NPM"
          img="https://spider.vn/images/icon_npmjs.svg"
        />

        <CardShortcut
          href="//yarnpkg.com"
          title="Yarn"
          img="https://spider.vn/images/icon_yarn.svg"
        />

        <CardShortcut
          href="//gruntjs.com"
          title="Grunt"
          img="https://spider.vn/images/icon_gruntjs.svg"
        />

        <CardShortcut
          href="//gulpjs.com"
          title="Gulp"
          img="https://spider.vn/images/icon_gulp.svg"
        />

        <CardShortcut
          href="//parceljs.org"
          title="Parcel"
          img="https://spider.vn/images/icon_parcel.png"
        />

        <CardShortcut
          href="//webpack.js.org"
          title="Webpack"
          img="https://spider.vn/images/icon_webpack.svg"
        />

        <CardShortcut
          href="//eslint.org"
          title="ESLint"
          img="https://spider.vn/images/icon_eslint.svg"
          alt="ESLint"
        />

        <CardShortcut
          href="//istanbul.js.org"
          title="Istanbul"
          img="https://spider.vn/images/icon_istanbul.png"
        />

        <CardShortcut
          href="//prettier.io"
          title="Prettier"
          img="https://spider.vn/images/icon_prettier.png"
        />

        <CardShortcut
          href="//git-scm.com"
          title="Git"
          img="https://spider.vn/images/icon_git.svg"
        />

        <CardShortcut
          href="//www.docker.com"
          title="Docker"
          img="https://spider.vn/images/icon_docker.svg"
        />
      </div>
    );
  }
}

export default Card;
