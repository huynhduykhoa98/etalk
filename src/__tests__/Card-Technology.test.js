/* eslint-disable linebreak-style */
import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Card from '../components/Technology/Card/Card';
import Teachnology from '../components/Technology/Technology';

Enzyme.configure({ adapter: new Adapter() });
it('test render', () => {
  const wrapped = shallow(<Teachnology />);
  expect(wrapped.find(Card).length).toEqual(1);
});
