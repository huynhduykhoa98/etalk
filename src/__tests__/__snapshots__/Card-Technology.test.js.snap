// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<div
  className="pic"
>
  <a
    href="//www.w3.org/standards/webdesign/htmlcss"
    title="GSAP"
  >
    <img
      alt="GSAP"
      src="https://spider.vn/images/icon_gsap.svg"
    />
    GSAP
  </a>
  <a
    href="//www.w3.org/standards/webdesign/htmlcss"
    title="HTML & CSS"
  >
    <img
      alt="HTML & CSS"
      src="https://spider.vn/images/icon_html.png"
    />
    HTML & CSS
  </a>
  <a
    href="//www.w3.org/standards/webdesign/script"
    title="JavaScript"
  >
    <img
      alt="JavaScript"
      src="https://spider.vn/images/icon_js.png"
    />
    JavaScript
  </a>
  <a
    href="//babeljs.io"
    title="Babel"
  >
    <img
      alt="Babel"
      src="https://spider.vn/images/icon_babel.svg"
    />
    Babel
  </a>
  <a
    href="//pugjs.org"
    title="Pug"
  >
    <img
      alt="Pug"
      src="https://spider.vn/images/icon_pugjs.svg"
    />
    Pug
  </a>
  <a
    href="//sass-lang.com"
    title="Sass"
  >
    <img
      alt="Sass"
      src="https://spider.vn/images/icon_sass.svg"
    />
    Sass
  </a>
  <a
    href="//getbootstrap.com"
    title="Bootstrap"
  >
    <img
      alt="Bootstrap"
      src="https://spider.vn/images/icon_bootstrap.png"
    />
    Bootstrap
  </a>
  <a
    href="//material.io"
    title="Material"
  >
    <img
      alt="Material"
      src="https://spider.vn/images/icon_material.png"
    />
    Material
  </a>
  <a
    href="//jquery.com"
    title="jQuery"
  >
    <img
      alt="jQuery"
      src="https://spider.vn/images/icon_jquery.svg"
    />
    jQuery
  </a>
  <a
    href="//nextjs.org"
    title="Next.js"
  >
    <img
      alt="Next.js"
      src="https://spider.vn/images/icon_nextjs.png"
    />
    Next.js
  </a>
  <a
    href="//angular.io"
    title="Angular"
  >
    <img
      alt="Angular"
      src="https://spider.vn/images/icon_angular.png"
    />
    Angular
  </a>
  <a
    href="//reactjs.org"
    title="React"
  >
    <img
      alt="React"
      src="https://spider.vn/images/icon_react.svg"
    />
    React
  </a>
  <a
    href="//redux.js.org"
    title="Redux"
  >
    <img
      alt="Redux"
      src="https://spider.vn/images/icon_redux.svg"
    />
    Redux
  </a>
  <a
    href="//graphql.org"
    title="GraphQL"
  >
    <img
      alt="GraphQL"
      src="https://spider.vn/images/icon_graphql.svg"
    />
    GraphQL
  </a>
  <a
    href="//vuejs.org"
    title="Vue"
  >
    <img
      alt="Vue"
      src="https://spider.vn/images/icon_vue.png"
    />
    Vue
  </a>
  <a
    href="//lodash.com"
    title="Lodash"
  >
    <img
      alt="Lodash"
      src="https://spider.vn/images/icon_lodash.svg"
    />
    Lodash
  </a>
  <a
    href="//underscorejs.org"
    title="Underscore"
  >
    <img
      alt="Underscore"
      src="https://spider.vn/images/icon_underscore.png"
    />
    Underscore
  </a>
  <a
    href="//backbonejs.org"
    title="Backbone.js"
  >
    <img
      alt="Backbone.js"
      src="https://spider.vn/images/icon_backbone.png"
    />
    Backbone.js
  </a>
  <a
    href="//emberjs.com"
    title="Ember.js"
  >
    <img
      alt="Ember.js"
      src="https://spider.vn/images/icon_ember.png"
    />
    Ember.js
  </a>
  <a
    href="//phaser.io"
    title="Phaser"
  >
    <img
      alt="Phaser"
      src="https://spider.vn/images/icon_phaser.png"
    />
    Phaser
  </a>
  <a
    href="//threejs.org"
    title="Three.js"
  >
    <img
      alt="Three.js"
      src="https://spider.vn/images/icon_threejs.png"
    />
    Three.js
  </a>
  <a
    href="//d3js.org"
    title="D3.js"
  >
    <img
      alt="D3.js"
      src="https://spider.vn/images/icon_d3.png"
    />
    D3.js
  </a>
  <a
    href="//www.gatsbyjs.org"
    title="Gatsby"
  >
    <img
      alt="Gatsby"
      src="https://spider.vn/images/icon_gatsby.svg"
    />
    Gatsby
  </a>
  <a
    href="//www.webcomponents.org"
    title="Web Components"
  >
    <img
      alt="Web Components"
      src="https://spider.vn/images/icon_web_components.svg"
    />
    Web Components
  </a>
  <a
    href="//www.polymer-project.org"
    title="Polymer"
  >
    <img
      alt="Polymer"
      src="https://spider.vn/images/icon_polymer.svg"
    />
    Polymer
  </a>
  <a
    href="//developers.google.com/web/progressive-web-apps"
    title="Progressive Web Apps"
  >
    <img
      alt="Progressive Web Apps"
      src="https://spider.vn/images/icon_web_app.png"
    />
    Progressive Web Apps
  </a>
  <a
    href="//serverless.css-tricks.com"
    title="Serverless"
  >
    <img
      alt="Serverless"
      src="https://spider.vn/images/icon_serverless.svg"
    />
    Serverless
  </a>
  <a
    href="//headlesscms.org/"
    title="Headless CMS"
  >
    <img
      alt="Headless CMS"
      src="undefined"
    />
    Headless CMS
  </a>
  <a
    href="//www.php.net"
    title="PHP"
  >
    <img
      alt="PHP"
      src="https://spider.vn/images/icon_php.svg"
    />
    PHP
  </a>
  <a
    href="//laravel.com"
    title="Laravel"
  >
    <img
      alt="Laravel"
      src="https://spider.vn/images/icon_laravel.svg"
    />
    Laravel
  </a>
  <a
    href="//octobercms.com"
    title="October"
  >
    <img
      alt="October"
      src="https://spider.vn/images/icon_october.png"
    />
    October
  </a>
  <a
    href="//symfony.com"
    title="Symfony"
  >
    <img
      alt="Symfony"
      src="https://spider.vn/images/icon_polymer.svg"
    />
    Symfony
  </a>
  <a
    href="//www.yiiframework.com"
    title="Yii"
  >
    <img
      alt="Yii"
      src="https://spider.vn/images/icon_yii.svg"
    />
    Yii
  </a>
  <a
    href="//www.codeigniter.com"
    title="CodeIgniter"
  >
    <img
      alt="CodeIgniter"
      src="https://spider.vn/images/icon_codeigniter.svg"
    />
    CodeIgniter
  </a>
  <a
    href="//wordpress.org"
    title="WordPress"
  >
    <img
      alt="WordPress"
      src="https://spider.vn/images/icon_wordpress.svg"
    />
    WordPress
  </a>
  <a
    href="//woocommerce.com"
    title="WooCommerce"
  >
    <img
      alt="WooCommerce"
      src="https://spider.vn/images/icon_woocommerce.svg"
    />
    WooCommerce
  </a>
  <a
    href="//www.drupal.org"
    title="Drupal"
  >
    <img
      alt="Drupal"
      src="https://spider.vn/images/icon_drupal.svg"
    />
    Drupal
  </a>
  <a
    href="//golang.org"
    title="Golang"
  >
    <img
      alt="Golang"
      src="https://spider.vn/images/icon_golang.svg"
    />
    Golang
  </a>
  <a
    href="//cordova.apache.org"
    title="Cordova"
  >
    <img
      alt="Cordova"
      src="https://spider.vn/images/icon_cordova.svg"
    />
    Cordova
  </a>
  <a
    href="//ionicframework.com"
    title="Ionic"
  >
    <img
      alt="Ionic"
      src="https://spider.vn/images/icon_ionic.svg"
    />
    Ionic
  </a>
  <a
    href="//facebook.github.io/react-native"
    title="React Native"
  >
    <img
      alt="React Native"
      src="https://spider.vn/images/icon_react.svg"
    />
    React Native
  </a>
  <a
    href="//www.nativescript.org"
    title="NativeScript"
  >
    <img
      alt="NativeScript"
      src="https://spider.vn/images/icon_nativescript.svg"
    />
    NativeScript
  </a>
  <a
    href="//flutter.dev"
    title="Flutter"
  >
    <img
      alt="Flutter"
      src="https://spider.vn/images/icon_flutterio.svg"
    />
    Flutter
  </a>
  <a
    href="//electronjs.org"
    title="Electron"
  >
    <img
      alt="Electron"
      src="https://spider.vn/images/icon_electronjs.svg"
    />
    Electron
  </a>
  <a
    href="//nodejs.org"
    title="Node.js"
  >
    <img
      alt="Node.js"
      src="https://spider.vn/images/icon_nodejs.svg"
    />
    Node.js
  </a>
  <a
    href="//expressjs.com"
    title="Express"
  >
    <img
      alt="Express"
      src="undefined"
    />
    Express
  </a>
  <a
    href="//hapi.dev"
    title="Hapi"
  >
    <img
      alt="Hapi"
      src="https://spider.vn/images/icon_hapi.svg"
    />
    Hapi
  </a>
  <a
    href="//sailsjs.com"
    title="Sails"
  >
    <img
      alt="Sails"
      src="https://spider.vn/images/icon_sails.svg"
    />
    Sails
  </a>
  <a
    href="//sequelize.org"
    title="Sequelize"
  >
    <img
      alt="Sequelize"
      src="https://spider.vn/images/icon_sequelizejs.svg"
    />
    Sequelize
  </a>
  <a
    href="//www.meteor.com"
    title="Meteor"
  >
    <img
      alt="Meteor"
      src="https://spider.vn/images/icon_meteor.svg"
    />
    Meteor
  </a>
  <a
    href="//www.sqlite.org"
    title="SQLite"
  >
    <img
      alt="SQLite"
      src="https://spider.vn/images/icon_sqlite.svg"
    />
    SQLite
  </a>
  <a
    href="//www.mysql.com"
    title="MySQL"
  >
    <img
      alt="MySQL"
      src="https://spider.vn/images/icon_mysql.svg"
    />
    MySQL
  </a>
  <a
    href="//www.postgresql.org"
    title="PostgreSQL"
  >
    <img
      alt="PostgreSQL"
      src="https://spider.vn/images/icon_postgresql.svg"
    />
    PostgreSQL
  </a>
  <a
    href="//www.mongodb.com"
    title="MongoDB"
  >
    <img
      alt="MongoDB"
      src="https://spider.vn/images/icon_mongodb.svg"
    />
    MongoDB
  </a>
  <a
    href="//redis.io"
    title="Redis"
  >
    <img
      alt="Redis"
      src="https://spider.vn/images/icon_redis.svg"
    />
    Redis
  </a>
  <a
    href="//www.npmjs.com"
    title="NPM"
  >
    <img
      alt="NPM"
      src="https://spider.vn/images/icon_npmjs.svg"
    />
    NPM
  </a>
  <a
    href="//yarnpkg.com"
    title="Yarn"
  >
    <img
      alt="Yarn"
      src="https://spider.vn/images/icon_yarn.svg"
    />
    Yarn
  </a>
  <a
    href="//gruntjs.com"
    title="Grunt"
  >
    <img
      alt="Grunt"
      src="https://spider.vn/images/icon_gruntjs.svg"
    />
    Grunt
  </a>
  <a
    href="//gulpjs.com"
    title="Gulp"
  >
    <img
      alt="Gulp"
      src="https://spider.vn/images/icon_gulp.svg"
    />
    Gulp
  </a>
  <a
    href="//parceljs.org"
    title="Parcel"
  >
    <img
      alt="Parcel"
      src="https://spider.vn/images/icon_parcel.png"
    />
    Parcel
  </a>
  <a
    href="//webpack.js.org"
    title="Webpack"
  >
    <img
      alt="Webpack"
      src="https://spider.vn/images/icon_webpack.svg"
    />
    Webpack
  </a>
  <a
    href="//eslint.org"
    title="ESLint"
  >
    <img
      alt="ESLint"
      src="https://spider.vn/images/icon_eslint.svg"
    />
    ESLint
  </a>
  <a
    href="//istanbul.js.org"
    title="Istanbul"
  >
    <img
      alt="Istanbul"
      src="https://spider.vn/images/icon_istanbul.png"
    />
    Istanbul
  </a>
  <a
    href="//prettier.io"
    title="Prettier"
  >
    <img
      alt="Prettier"
      src="https://spider.vn/images/icon_prettier.png"
    />
    Prettier
  </a>
  <a
    href="//git-scm.com"
    title="Git"
  >
    <img
      alt="Git"
      src="https://spider.vn/images/icon_git.svg"
    />
    Git
  </a>
  <a
    href="//www.docker.com"
    title="Docker"
  >
    <img
      alt="Docker"
      src="https://spider.vn/images/icon_docker.svg"
    />
    Docker
  </a>
</div>
`;
